pkg/driver/driver.go

```diff
@@ -40,37 +40,43 @@ const (
 )
 
 const (
-	DriverName  = "ebs.csi.aws.com"
-	TopologyKey = "topology." + DriverName + "/zone"
+	// Name is the name of the driver.
+	Name = "ebs.csi.aws.com"
+	// Topology is the zone topology key used by the driver.
+	TopologyKey = "topology." + Name + "/zone"
 )
 
+// Driver is a gRPC server for both the controller and the node CSI services.
 type Driver struct {
 	controllerService
 	nodeService
 
 	srv     *grpc.Server
-	options *DriverOptions
+	options *Options
 }
 
-type DriverOptions struct {
-	endpoint        string
-	extraVolumeTags map[string]string
-	mode            Mode
+// Options contains configuration options for the driver.
+type Options struct {
```

Comment:
Just checking was `DriverOptions` to `Options` change made to make `govet` etc happy?  having this change included in the same commit makes it harder to review the actual code that this PR proposes. If you really want to make this change then please do that in a different commit.